README.md

```diff
@@ -1,7 +1,7 @@
 # Remote-Collection
-## A TypeScript `RemoteData` Collection class
+A TypeScript `RemoteData` Collection class
 
-### Rationale
+<hr />
 
 There have been a number of articles written about using Algebraic Data Types to
 describe the states that data fetching entails. Replacing all of the
```

Comment:
`ergonomics patterns` or `ergonomic patterns`? the first would be correct if you're talking about patterns of ergonomics.

Reply:
⚡️ https://github.com/ca-la/remote-collection/pull/11/commits/a3abd52a4a55db79e66f84210e283d2ec17cea9a

Cleaned the opening paragraph up a bit to address the two (excellent) clarification requests here.

Reply:
And also https://github.com/ca-la/remote-collection/pull/11/commits/f833bab3f83995cff4d6c529d1cad828d4c95bb3